### PR TITLE
Some more Chemical things

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Sprite by Sheeple
Adds currently admin only chemicals

### CODE REVIEW

File: code/modules/reagents/machinery/alchemy.dm [hunk 1]
```diff
@@ -0,0 +1,89 @@
+//Alchemy!
+
+/obj/structure/alchemy
+	name = "Ambelic"
+	desc = "A older type of chemical bonding and condencing into different liquids, how its made is lost in time, yet somehow still stands here today."
```
Comment (ThePainkiller) [inline code/modules/reagents/machinery/alchemy.dm:5]:
```suggestion
	desc = "An archaic type of alchemical still, bonding and condensing different liquids together. Knowledge on how it synthesizes materials has been lost to time, yet it somehow can still be used by the chemistry savvy."
```

File: code/modules/reagents/machinery/alchemy.dm [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		if(S.amount >= 1)
+			catalists = new I(amount = 1)
+			S.use(1)
+			to_chat(user, "You add a single sheet of [S] to the flask base.")
+		else
+			to_chat(user, "You need at lest a single sheet of [src] to add to the flask base.")
+*/
+	var/obj/item/reagent_containers/B = I
+	if(beaker)
+		to_chat(user, "A fitted beaker is already loaded.")
```
Comment (ThePainkiller) [inline code/modules/reagents/machinery/alchemy.dm:69]:
```suggestion
		to_chat(user, "A fitting beaker is already loaded.")
```

File: code/modules/reagents/machinery/alchemy.dm [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		return
+	if(istype(B, /obj/item/reagent_containers/glass))
+		if(!istype(B, /obj/item/reagent_containers/glass/beaker/flask_alchemy))
+			to_chat(user, SPAN_NOTICE("This only seems to take a really fancy flask."))
```
Comment (ThePainkiller) [inline code/modules/reagents/machinery/alchemy.dm:73]:
```suggestion
			to_chat(user, SPAN_NOTICE("This only seems to take a really fancy type of flask."))
```

File: code/modules/reagents/machinery/alchemy.dm [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+			return
+		src.beaker =  B
+		if (user.unEquip(B, src))
+			to_chat(user, "You set the flask onto the [src].")
```
Comment (ThePainkiller) [inline code/modules/reagents/machinery/alchemy.dm:77]:
```suggestion
			to_chat(user, "You insert the flask into the [src].")
```

File: code/modules/reagents/machinery/alchemy.dm [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+			update_icon()
+			return
+
+/obj/item/reagent_containers/glass/beaker/flask_alchemy
+	name = "ambelic flask"
+	desc = "A thick glass bulb like bottle that can not hold much liquid and dosnt seem to allow any mixing or reacting."
```
Comment (ThePainkiller) [inline code/modules/reagents/machinery/alchemy.dm:83]:
```suggestion
	desc = "A thick, bulb-like glass bottle that can't hold much liquid and doesn't seem to allow any mixing or reacting within."
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 6]
```diff
@@ -0,0 +1,225 @@
+/datum/reagent/iron_skin_brew
+	name = "Fe Na of NaCl"
+	id = "iron_skin_brew"
+	description = "A mix of metals and salts boiled into one a other said to make people able to take more physical damage but being of metal makes you weak to fire. Also makes breathing harder..."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:4]:
```suggestion
	description = "A mix of metals and salts boiled into one another, said to be able to make people take more physical punishment, with the drawback of making your metallic coating more vulnerable to fire. It also makes breathing harder..."
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+	ironskin.falls_mod += 0.4
+	ironskin.Paralyse(3)
+	ironskin.burn_mod_perk -= 0.4
+	ironskin.brute_mod_perk += 0.2
+	ironskin.oxy_mod_perk  -= 3
+
+/datum/reagent/toxin_draft
+	name = "Toxic Draft"
+	id = "toxin_draft"
+	description = " A horrable brew that is toxic to almost all beings."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:48]:
```suggestion
	description = " A horrible brew that is toxic to almost all beings."
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 8]
```diff
@@ -0,0 +1,225 @@
+/datum/reagent/iron_skin_brew
+	name = "Fe Na of NaCl"
+	id = "iron_skin_brew"
+	description = "A mix of metals and salts boiled into one a other said to make people able to take more physical damage but being of metal makes you weak to fire. Also makes breathing harder..."
+	taste_description = "salt of metal"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:5]:
```suggestion
	taste_description = "salt with a metallic aftertaste"
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+		var/obj/item/organ/internal/eyes/E = H.random_organ_by_process(OP_EYES)
+		if(E && istype(E))
+			if(E.damage > 0)
+				E.damage = max(E.damage - (5), 0)
+
+
+/datum/reagent/lively_concoxion
+	name = "Ichor of Health"
+	id = "lively_concoxion"
+	description = "A jellie of blood said to make whomever eats this to have a more perfected body. Untill it proccesses out."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:107]:
```suggestion
	description = "A jellyfied blood-like toxic compound said to make whomever eats it to have a more perfected body... Untill it proccesses out."
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+	taste_description = "dry iron"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:108]:
```suggestion
	taste_description = "metallic gelatin"
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+	M.maxHealth -= 50
+	M.health -= 50
+	M.adjustOxyLoss(60)
+	M.Weaken(5)
+	M.silent = max(M.silent, 5)
+
+/datum/reagent/nervs
+	name = "Nervs"
+	id = "nervs"
+	description = "A uniquic mix that makes the users able to withstand more chemicals."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:141]:
```suggestion
	description = "A unique mix that makes the users' brain able to withstand more chemical accumulation in their system."
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+
+/datum/reagent/nervs/on_mob_delete(mob/living/carbon/M)
+	. = ..()
+	var/mob/living/carbon/C = M
+	if(istype(C))
+		C.metabolism_effects.nsa_threshold = initial(C.metabolism_effects.nsa_chem_bonus)
+		C.metabolism_effects.calculate_nsa()
+
+/datum/reagent/harms
+	name = "Painful Powerders"
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:172]:
```suggestion
	name = "Painful Powder"
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+	id = "harms"
+	description = "A mix of soild sharp sands designed to cut down who ever they are thrown at."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:174]:
```suggestion
	description = "A mix of solid sharp sands designed to cut down whoever they are thrown at." // POCKET SAND!
```

File: code/modules/reagents/reagents/alchemy.dm [hunk 14]
```diff
@@ -0,0 +1,225 @@
...
+	. = ..()
+	if(!ishuman(L))
+		L.damage_through_armor(230, BRUTE, attack_flag = ARMOR_BIO)
+		return
+	L.damage_through_armor(20, BRUTE, attack_flag = ARMOR_BIO)
+
+/datum/reagent/burns
+	name = "Burning Oils"
+	id = "burns"
+	description = "Heated oils with fibers of iron and salt. Well its cold to the glass to your skin it burns..."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagents/alchemy.dm:202]:
```suggestion
	description = "Heated oils with fibers of iron and salt. While it's cold inside the glass, contact with skin, chitin or any dermis tissue makes its temperature violently react..."
```

File: code/modules/reagents/recipes.dm [hunk 15]
```diff
@@ -49,6 +49,7 @@
 	var/result = null
 	var/list/required_reagents = list()
 	var/list/catalysts = list()
+	var/list/alchemy_catalysts = list()
```
Comment (ThePainkiller) [inline code/modules/reagents/recipes.dm:52]: Is this var used for anything?

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Some minor typo fixes, as usual

File: code/modules/reagents/recipes.dm (hunk 15 above)
Comment (Trilbyspaceclone) [reply code/modules/reagents/recipes.dm:52]: Nope that idea was scrapped after a few days of tweaking to no avail on it working

**Comment by cdb-is-not-good** [comment]
Please make sure your changelog is fully detailed- also is this more weird magic...?

**Pull request merged**